functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -61,7 +67,8 @@ public BuildConfigurator withRemoteDebugging() {
     }
 
     /**
-     * @param port if set to BuildConfigurator.RANDOM_PORT it will take random, free port
+     * @param port
+     *     if set to BuildConfigurator.RANDOM_PORT it will take random, free port
```

Comment:
please, format only those lines that are changed by you... `Only VCS changed text`

Reply:
Done